frontend/src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,43 @@
+.logo {
```

Comment:
It's ok to use .css in this if that's what you're comfortable with, but it might get a little wonky if the navbar uses .css and the rest of the project uses .styles.js files. I'd encourage you to take the time to convert this to a styled-component for the navbar, if you think you have the time to do so! 

You can do something like 
`export const Navbar = styled.nav`  or `styled.div` even and then import the NavBar component into the NavBar.jsx and use it just like a `<nav>` tag with all of the styles.